builtin/providers/circonus/resource_circonus_check_httptrap_test.go

```diff
@@ -86,7 +86,7 @@ resource "circonus_check" "consul" {
   period = "60s"
 
   collector {
-    id = "/broker/1286"
+    id = "/broker/2110"
```

Comment:
Probably unrelated to this PR, but where does this path and `consul-server-10-151-2-8` come from and what's the significance of these? i.e. would Circonus accept any hostname & path? 🤔